repository-meta-analyzer/src/main/java/org/hyades/model/IntegrityModel.java

```diff
@@ -0,0 +1,66 @@
+package org.hyades.model;
+
+import org.hyades.persistence.model.Component;
+import org.hyades.proto.repointegrityanalysis.v1.HashMatchStatus;
+
+import java.io.Serializable;
+import java.util.Date;
+
+public class IntegrityModel implements Serializable {
+
+    private Component component;
+    private Date publishedTimestamp;
+    private HashMatchStatus md5HashMatched;
```

Comment:
Very minor: variables like md5HashMatchStatus would be more readable here. 
Method name isMd5HashMatched looks like should be returning boolean. getMd5HashMatchStatus might be more readable